app/package.json

```diff
@@ -18,24 +18,24 @@
     "@onaio/session-reducer": "^0.0.12",
     "@onaio/utils": "^0.0.1",
     "@opensrp/ant-icons": "^0.0.1",
-    "@opensrp/card-support": "^0.0.6",
+    "@opensrp/card-support": "^0.0.7",
     "@opensrp/error-boundary-fallback": "^0.0.5",
     "@opensrp/form-config-antd": "^0.0.2",
-    "@opensrp/inventory": "^0.0.6",
+    "@opensrp/inventory": "^0.0.7",
     "@opensrp/keycloak-service": "^0.0.15",
-    "@opensrp/location-management": "^0.0.12",
+    "@opensrp/location-management": "^0.0.13",
     "@opensrp/notifications": "^0.0.5",
     "@opensrp/pkg-config": "^0.0.9",
-    "@opensrp/plans": "^0.0.7",
+    "@opensrp/plans": "^0.0.8",
     "@opensrp/product-catalogue": "^0.0.7",
     "@opensrp/react-utils": "^0.0.10",
     "@opensrp/reducer-factory": "^0.0.13",
     "@opensrp/server-logout": "^0.0.1",
     "@opensrp/server-service": "^0.0.16",
     "@opensrp/store": "^0.0.10",
-    "@opensrp/team-assignment": "^0.0.3",
-    "@opensrp/team-management": "^0.0.6",
-    "@opensrp/user-management": "^0.1.15",
+    "@opensrp/team-assignment": "^0.0.4",
+    "@opensrp/team-management": "^0.0.7",
+    "@opensrp/user-management": "^0.1.16",
     "@sentry/react": "^5.27.4",
     "add": "^2.0.6",
```

Comment:
Lets remove this one ? i am sure it was added mistakenly ?

Reply:
cool.